main.tf

```
@@ -429,75 +429,71 @@ resource "aws_config_config_rule" "iam_user_active" {
     }
   }
 
-  depends_on = [
-    "aws_lambda_permission.iam_user_active",
-  ]
+  depends_on = [aws_lambda_permission.iam_user_active]
 }
 
 ### config_enabled
 module "lambda_config_enabled" {
-  source = "git::https://github.com/plus3it/terraform-aws-lambda.git?ref=v0.12.0"
+  source = "git::https://github.com/plus3it/terraform-aws-lambda.git?ref=v1.1.0"
 
   function_name = "config_rule_config_enabled"
   description   = "Checks that Config has been activated and is logging to a specific bucket and sending to a specifc SNS topic"
   handler       = "config_enabled.lambda_handler"
   runtime       = "python3.6"
   timeout       = 15
-  tags          = "${var.tags}"
+  tags          = var.tags
 
   reserved_concurrent_executions = "-1"
 
   source_path = "${local.aws_config_rules}/python/config_enabled.py"
 
-  attach_policy = "${var.create_config_rules && ! local.exclude_config_enabled}"
-  policy        = "${join("", data.aws_iam_policy.config_rules.*.policy)}"
+  policy = var.create_config_rules ? { json = data.aws_iam_policy.config_rules[0].policy } : null
```

This might have been an oversight before, but i think this resource should have a corresponding `aws_iam_policy_document` like all the others...
```suggestion
  policy = var.create_config_rules ? data.aws_iam_policy_document.config_enabled[0] : null
```

---

Isn't the purpose of this one being an `aws_iam_policy` so that the policy is managed by AWS and therefore it's one less thing that has to be managed?

---

All of the others use `aws_iam_policy` also, just filtered through `aws_iam_policy_document`. something like this should work.

```
data "aws_iam_policy_document" "lambda_config_enabled" {
  count = "${var.create_config_rules && ! local.exclude_config_enabled ? 1 : 0}"

  source_json = "${data.aws_iam_policy.config_rules.policy}"
```

---

Fixed